.github/workflows/neon_extra_builds.yml

```diff
@@ -338,7 +347,7 @@ jobs:
       contains(github.event.pull_request.labels.*.name, 'run-extra-build-stats') ||
       contains(github.event.pull_request.labels.*.name, 'run-extra-build-*') ||
       github.ref_name == 'main'
-    runs-on: [ self-hosted, gen3, large ]
+    runs-on: [ self-hosted, large, x64 ]
```

Comment:
Is this a new label for the current x86-64 runners we have? I'd either update it in all the places or keep it as is

Reply:
yes, `x64` and `arm64` are labels inherited automatically from uname info (and `Linux` as well)